### PR TITLE
fix: show error if create playground/service actions fail

### LINKED ISSUE #334: Playground start error icon clickable but NOP
When an error is reported to the playground, an icon is displayed right, it is clickable but nothing is happening when clicking on the icon

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

It shows the error message if the create playground action fails and re-enable the button

### Screenshot / video of UI

show_error_create_playground

### What issues does this PR fix or reference?

it fixes #334 

### How to test this PR?

1. run test or make the `requestCreatePlayground` fails somehow (e.g update code to throw) and see if the error is displayed and you're able to click again the button

### CODE REVIEW

File: packages/frontend/src/pages/PlaygroundCreate.svelte [hunk 1]
```diff
@@ -178,6 +182,9 @@ onDestroy(() => {
             placeholder="Optionally provide system prompt to define general context, instructions or guidelines to be used with each query"
           ></textarea>
         </div>
+        {#if errorMsg}
+          <div aria-label="Error message" class="text-red-500 text-sm">{errorMsg}</div>
+        {/if}
```
Comment (benoitf) [inline packages/frontend/src/pages/PlaygroundCreate.svelte:187]: Maybe use https://github.com/containers/podman-desktop-extension-ai-lab/blob/main/packages/frontend/src/lib/ErrorMessage.svelte ?

**Comment by axel7083** [comment]
Should this also be repeated in the CreateInferenceServer component ?

**Comment by lstocchi** [comment]
> Should this also be repeated in the CreateInferenceServer component ?

Checking

**Comment by lstocchi** [comment]
> Should this also be repeated in the CreateInferenceServer component ?

Fixed there as well. Thanks for pointing it out!! 👍

File: packages/frontend/src/pages/PlaygroundCreate.svelte [hunk 2]
```diff
@@ -21,6 +22,7 @@ let modelId: string | undefined = undefined;
 let systemPrompt: string | undefined = undefined;
 let submitted: boolean = false;
 let playgroundName: string;
+let errorMsg = '';
```
Comment (axel7083) [inline packages/frontend/src/pages/PlaygroundCreate.svelte:25]:
```suggestion
let errorMsg: string | undefined = undefined;
```

File: packages/frontend/src/pages/PlaygroundCreate.svelte [hunk 3]
```diff
@@ -51,6 +53,7 @@ function onNameInput(event: Event) {
 }
 
 async function submit() {
+  errorMsg = '';
```
Comment (axel7083) [inline packages/frontend/src/pages/PlaygroundCreate.svelte:56]:
```suggestion
  errorMsg = undefined;
```

File: packages/frontend/src/pages/CreateService.svelte [hunk 4]
```diff
@@ -21,6 +22,8 @@ $: localModels = $modelsInfo.filter(model => model.file);
 let containerPort: number | undefined = undefined;
 // The modelId is the bind value to form input
 let modelId: string | undefined = undefined;
+// If the creation of a new inference service fail
+let errorMsg = '';
```
Comment (axel7083) [inline packages/frontend/src/pages/CreateService.svelte:26]:
```suggestion
let errorMsg: string | undefined = undefined;
```

File: packages/frontend/src/pages/CreateService.svelte [hunk 5]
```diff
@@ -162,6 +167,9 @@ onMount(async () => {
             name="containerPort"
             required />
         </div>
+        {#if errorMsg}
```
Comment (axel7083) [inline packages/frontend/src/pages/CreateService.svelte:170]:
```suggestion
        {#if errorMsg !== undefined}
```

**Review by axel7083 — APPROVED** [review]

**Pull request merged**